### PR TITLE
Wurzel change prestige name and parameters

### LINKED ISSUE #1141: Rename fn_prestige.sqf - current name is unclear
*Version:* all?

***Issue:***

This function is adding aggresion to factions. The last name I have for it is prestige. Something like fn_addAggo/fn_addAggression would sound much better.

### PR DESCRIPTION
**Pull request by wurzel0701**
## What type of PR is this.
1. [ ] Bug
2. [X] Change
3. [ ] Enhancement

### What have you changed and why?
Information: The name prestige.sqf was not referring to the actual usage of the file, its now called addAggression, whichs reflects the function nicely.

The input parameter has been modified after Johns comments about it
[[aggroOcc, timeOcc],[aggroInv, timeInv]] => [side, aggro, time]

This made the function easier to use and reduced the number of calls in most cases.

This will solve #1141 making the previously set up PR obsolete, therefor I will close the old PR.

Untested so far, gonna make sure it dont die on start in a second, needs a large test to make sure it has no typos.

### Please specify which Issue this PR Resolves.
closes #1141

### Please verify the following and ensure all checks are completed.

1. [ ] Have you loaded the mission in LAN host?
2. [ ] Have you loaded the mission on a dedicated server?

### Is further testing or are further changes required?
1. [ ] No
2. [X] Yes (Please provide further detail below.)

### How can the changes be tested?
Steps: Play the game and get as many different aggro change events as possible

********************************************************
Notes:

### CODE REVIEW

File: A3-Antistasi/functions/Base/fn_addAggression.sqf [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    ["_aggroChange", 0, [0]],
+    ["_aggroTime", 0, [0]]
+];
+
+_fn_convertMinutesToDecayRate =
+{
+    params ["_points", "_minutes"];
+    if(_minutes == 0) then
+    {
+        [1, "Minute parameter is 0, assuming 1", "addAggression"] call A3A_fnc_log;
```
Comment (HakonRydland) [inline A3-Antistasi/functions/Base/fn_addAggression.sqf:39]: arg!!! new macros
at topp
```sqf
#include "..\..\Includes\LogMacros"
```

Logging
```suggestion
        Error("Minute parameter is 0, assuming1");
```

File: A3-Antistasi/functions/Base/fn_addAggression.sqf [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+        _minutes = 1;
+    };
+    private _decayRate = (-1) * (_points / _minutes);
+    _decayRate;
```
Comment (HakonRydland) [inline A3-Antistasi/functions/Base/fn_addAggression.sqf:43]: could just return it without storing it in a variable in between
```suggestion
    (-1) * (_points / _minutes);
```

File: A3-Antistasi/functions/Base/fn_addAggression.sqf [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+    <NUMBER> gameMode
+    <SIDE> Occupants
+    <ARRAY> aggressionStackOccupants
+    <SIDE> Invaders
+    <ARRAY> aggressionStackInvaders
+
+Example:
+[Occupants, 50, 15] call A3A_fnc_addAggression;
+*/
+
```
Comment (HakonRydland) [inline A3-Antistasi/functions/Base/fn_addAggression.sqf:26]:
```suggestion
#include "..\..\Includes\LogMacros"
```

**Review by HakonRydland — APPROVED** [review]
looks good, would recommend moving to new macros though. 😉

**Comment by Bob-Murphy** [comment]
merge conflicts and requested changes

**Comment by wurzel0701** [comment]
Merge conflicts cleared for now, better merge fast :)

**Comment by Bob-Murphy** [comment]
If ready to merge, please mark it as such.

**Pull request merged**